opensearch_py_ml/ml_commons/ml_common_utils.py

```diff
@@ -6,7 +6,7 @@
 # GitHub history for details.
 
 ML_BASE_URI = "/_plugins/_ml"
-MODEL_UPLOAD_CHUNK_SIZE = 10_000_000
+MODEL_REGISTER_CHUNK_SIZE = 10_000_000
```

Comment:
How about change to `MODEL_CHUNK_SIZE_LIMIT`

Reply:
Or to match the `MODEL_MAX_SIZE`, another option is `MODEL_CHUNK_MAX_SIZE` ?

Reply:
Done